src/cc_catalog_airflow/dags/provider_api_scripts/smithsonian.py

```diff
@@ -272,17 +274,27 @@ def _get_creator(row, creator_types=CREATOR_TYPES):
         ],
         key=lambda x: creator_types[x['label'].lower()]
     )
-    freetext_creator_generator = (
-        c['content'] for c in ordered_freetext_creator_objects
-    )
+
     indexed_structured_creator_generator = (
         i['content'] for i in _check_type(indexed_structured.get('name'), list)
         if type(i) == dict
         and _check_type(i.get('type'), str).lower() == 'personal_main'
         and _check_type(i.get('content'), str)
     )
 
-    creator = next(freetext_creator_generator, None)
+    if ordered_freetext_creator_objects:
+        c = ordered_freetext_creator_objects[0]
+        priority = creator_types[c['label'].lower()]
+
+        creators_list = [c['content'] for c in ordered_freetext_creator_objects
+                         if priority == creator_types[c['label'].lower()]]
+
+        creator = '; '.join(creators_list[:-1]) + ' and ' + creators_list[-1] \
```

Comment:
Did you mean to add a comma instead of a semicolon?
```suggestion
        creator = ', '.join(creators_list[:-1]) + ' and ' + creators_list[-1] \
```

Reply:
Thanks heaps for the comments @allen505 ! In fact I changed the delimiter from comma to semicolon since sometimes a single creator value too contains commas within the string. Using a semicolon is not ideal, however I'm unsure how else we can make a distinction between the seperate creator values and the comma separation within a single creator string.

Reply:
That's interesting. Do we know why a single creator value has commas within it?
If it's because of multiple creators, then do we need to make a distinction between the separate creator values and the comma separation within a single creator string?

Reply:
It's not necessarily multiple creators. For example creators appearing for the SIL unit code may look like 'Greenawald, John L' where the value coming before the comma is the surname.